scoping_document.md

```diff
@@ -52,21 +52,21 @@ App usage is divided into different phases, based on contact points and interact
+In this phase, a person decides to get more information about the app. In this phase, persons may have different questions regarding usage of the app (application, data privacy, accessibility, and so on). It should be possible to have these questions answered before downloading the app (hotline, information on the websites of the RKI and Federal Ministry of Health (BMG), App Store, Google Play Store).
 
 #### *Installation* phase
-A person decides to download the app (App Store/Google Play Store) and, after the technical installation process, is given an introduction to the app the first time it is opened. In this introductory phase, app users are given an overview of the functionality, terms of use, and privacy provisions; are asked for the necessary consent; and guided through the notification and other settings.
+A person decides to download the app (App Store or Google Play Store) and, after the technical installation process, is given an introduction to the app the first time it is opened. In this introductory phase, app users are given an overview of the functionality, terms of use, and privacy provisions; are asked for the necessary consent; and guided through the notification and other settings.
 
 #### *Usage* phase
 The usage phase is subdivided into four further areas, in which people who use the app have different needs.
 
 1. **Background**
 
-   In idle mode, the application runs in the background on the smartphone and saves the pseudo IDs of other nearby app users, automatically and encrypted. At regular intervals, the app pulls from the server a list of pseudo IDs of the persons who have voluntarily reported that they are infected. The app compares their pseudo IDs with those stored on the smartphone to determine whether there has been any exposure.
+   In idle mode, the app runs in the background on the smartphone and saves the pseudo IDs of other nearby app users, automatically and encrypted. At regular intervals, the app pulls from the server a list of pseudo IDs of the persons who have voluntarily reported that they are infected. The app compares their pseudo IDs with those stored on the smartphone to determine whether there has been any exposure.
 
 2. **Exposure**
 
-   If exposure to infected persons is determined, app user receives a notification and recommendations on what to do. For instance, that they should contact a medical professional or their responsible health department, and whether they should self-isolate.
+   If exposure to infected persons is determined, the app user receives a notification and a recommendations on what to do. For instance, the app user is asked to contact a medical professional or their responsible health department, or the app user is asked to self-isolate or the app user is notified that there is no need to self-isolate.
```

Comment:
>For instance, the app user is asked to contact a medical professional or their responsible health department, or the app >user is asked to self-isolate or the app user is notified that there is no need to self-isolate.

would, in my opinion, look better like:
> For instance, the user could be recommended to contact a medical professional or their responsible health department, or alternatively asked to self-isolate. The user can also be notified that there is no need to self-isolate.

Reply:
I ended up with to remove the passive voice and be consistent about the phrases "ask" and "recommend"

> For instance, the app recommends the user to contact a medical professional or their responsible health department, or alternatively, the app recommends the user to self-isolate. Contrarily, the app can also notify the user that there is no need to self-isolate.